packaging/nntrainer.spec

```diff
@@ -12,7 +12,7 @@
 %define         gen_input $(pwd)/test/input_gen/genInput.py
 %define         support_data_augmentation_opencv 1
 %define         configure_subplugin_install_path -Dnnstreamer-subplugin-install-path=%{nnstreamer_subplugin_path}
-
+%define         use_encoder 1
```

Comment:
We don't need to define use_encoder here.

Reply:
Thank you for Review - Fix